packages/suite/src/middlewares/wallet/fiatRatesMiddleware.ts

```diff
@@ -51,23 +51,19 @@ const fiatRatesMiddleware = (api: MiddlewareAPI<Dispatch, AppState>) => (next: D
         case ACCOUNT.CREATE: {
             // fetch current rates for account's tokens
             const account = action.payload;
-            if (account.tokens) {
-                account.tokens.forEach((t, i) => {
-                    if (t.symbol) {
-                        const s = t.symbol;
-                        // wait 500ms before firing next fetch
-                        setTimeout(() => {
-                            api.dispatch(
-                                fiatRatesActions.updateCurrentRates({
-                                    symbol: s,
-                                    mainNetworkSymbol: account.symbol,
-                                    tokenAddress: t.address,
-                                }),
-                            );
-                        }, i * 500);
-                    }
-                });
-            }
+            account.tokens?.forEach(t => {
+                if (!t.symbol) {
+                    return;
+                }
+                api.dispatch(
+                    fiatRatesActions.updateCurrentRates({
+                        symbol: t.symbol!,
```

Comment:
```suggestion
                        symbol: t.symbol,
```
On line 55, you checked that t.symbol is not undefined/null.

Maroš 23-05-2021 17:56 ` ja moc nemam rad tie !`

Reply:
good catch :) it is not necessary anymore. same as in `ACCOUNT.UPDATE` few lines above!